package.json

```diff
@@ -61,12 +61,12 @@
     "typescript": "^3.7.3"
   },
   "scripts": {
-    "compile": "hardhat compile",
-    "test": "hardhat test",
+    "compile": "hardhat compile && forge build",
+    "test:hardhat": "yarn compile && hardhat test",
     "test:gas": "UPDATE_SNAPSHOT=1 yarn test --grep gas",
-    "test:all": "UPDATE_SNAPSHOT=1 yarn test",
+    "test:all": "UPDATE_SNAPSHOT=1 yarn test:hardhat && forge test --isolate",
     "prettier:fix": "prettier --write '**/*.ts' && prettier --write '**/*.json'",
-    "lint:fix": "yarn prettier:fix && forge fmt",
-    "prettier": "prettier --check '**/*.ts' && forge fmt --check"
+    "lint": "yarn prettier:fix && forge fmt",
+    "lint:check": "prettier --check '**/*.ts' && forge fmt --check"
```

Comment:
renamed some commands here
`yarn test:all` wasnt running all the tests... only hardhat. So now `yarn test:hardhat` runs hardhat, and `yarn test:all` runs both
`yarn lint` should just fix everything locally both javascript and solidity then`yarn lint:check` can run the reverting commands which have to be run in CI